server/src/main/scala/actors/ProdDrtSystem.scala

```diff
@@ -57,10 +54,16 @@ case class ProdDrtSystem(config: Configuration, airportConfig: AirportConfig)
 
   override val aggregatedArrivalsActor: ActorRef = system.actorOf(Props(new AggregatedArrivalsActor(ArrivalTable(airportConfig.portCode, PostgresTables))), name = "aggregated-arrivals-actor")
 
-  override val liveCrunchStateActor: ActorRef = system.actorOf(liveCrunchStateProps, name = "crunch-live-state-actor")
-  override val forecastCrunchStateActor: ActorRef = system.actorOf(forecastCrunchStateProps, name = "crunch-forecast-state-actor")
+  val legacyFlightDataCutoff: SDateLike = SDate(config.get[String]("legacy-flight-data-cutoff"))
 
-  override val portStateActor: ActorRef = PortStateActor(now, liveCrunchStateActor, forecastCrunchStateActor, airportConfig.queuesByTerminal)
+  override val portStateActor: ActorRef = if (usePartitionedPortState) {
```

Comment:
lets try run these in parallel for a few days before we switch over so there is some overlap in the state storage.

Reply:
noted to create a migration service to migrate existing crunch & staff minutes to their daily counterparts before enabling the partitioned flag